doc/source/project/releases.rst

```diff
@@ -0,0 +1,70 @@
+======================
+Galaxy Release Process
+======================
+
+This document outlines the Galaxy release cycle so administrators and users know
+when to expect updates and what each update includes, making it easier to plan
+for upgrades and maintain their systems.  It does not cover the actual execution
+of the release process, which is covered elsewhere. (link)
+
+
+Major or Long Term Support (LTS) Releases
+-----------------------------------------
+
+The first release of each year is Galaxy's annual LTS release. The LTS release
+is a major version update corresponding to the year, for example, 24.0.  This
+release receives significant and thorough testing by a dedicated team that does
+end-to-end testing of a variety of analysis protocols and will be supported and
+receive bugfixes until the next LTS.  As such, this release is ideally targeted
+for production use by local Galaxy administrators.
+
+
+Minor Releases
+--------------
+
+Throughout the year, we also ship several (currently 2-3) minor releases. These
+are version updates like 24.1, 24.2.  These can also include significant
+updates, new features, etc., but do not follow the full release-testing protocol
+as with the LTS release.  These are targeted for administrators who want to stay
+up to date with the latest features and improvements.  Usegalaxy.org deploys new
+minor releases as a part of the release process, and other usegalaxy.* servers
+should strive to stay up to date with these.
+
+
+Point Releases
+--------------
+
+Point releases (e.g. 24.1.1) are issued to address bug fixes and security
+updates. These are not scheduled and are released as needed.  Following the
+release branch (i.e. release_24.0) is recommended to stay up to date with these. 
+
+
+Release Schedule
+----------------
+
+Our release schedule is as follows:
+
+- LTS Release: Annually, in the first quarter.
+- Minor Releases: 2-3 times a year, generally in the early summer and fall.
+- Point Releases: As needed.
+
+Please note that the release dates can vary based on the development process and
+testing results.
+
+The usegalaxy.* public servers will be updated to the latest release within a
```

Comment:
We should double check with EU/AU admins before we make such a promise.

Reply:
ping @bgruening @cat-bro

Reply:
xref https://galaxyproject.org/usegalaxy/#membership, which says 90 days -- we should make sure these align.